imports/importers.py

```diff
@@ -44,20 +44,24 @@ class ShapefileImporter:
         'pinta_ala': 'area',
     }
 
-    def __init__(self, zipped_shapefile):
```

Comment:
Here you are changing code that was written in the previous commit. I can see no reason to have both commits in the codebase and would suggest to merge these to commits into one.

Reply:
merged